docker/dockerhub_to_novi_ecr.sh

```diff
@@ -57,6 +57,7 @@ docker pull --disable-content-trust=false docker.io/libra/validator:"$DOCKERHUB_
 docker pull --disable-content-trust=false docker.io/libra/validator_tcb:"$DOCKERHUB_TAG"
 docker pull --disable-content-trust=false docker.io/libra/cluster_test:"$DOCKERHUB_TAG"
 docker pull --disable-content-trust=false docker.io/libra/client:"$DOCKERHUB_TAG"
+docker pull --disable-content-trust=false docker.io/vault:"1.4.0"
```

Comment:
Given this file has no hard coded version numbers I'm loath to have it take them on now.   At the very least this should be input for the file, at best there should be a single file that acts as the source of truth for the vault version, and given the sensitivity of the software perhaps a sha version pull as well, like in all our debian base versions in our docker builds.    But by having that single file perhaps RUNTIME_SERVICES, set up with things like vault=1.4.0@sha256:<sha> we could use it as input here to make sure we pull the exact file we want, and elsewhere use that version number as a way to reference it.